charts/gcp-marketplace/pxc-operator/README.md

```diff
@@ -0,0 +1,352 @@
+  --set pxc57backup.image.tag=${TAG} \
+  --set pxc.version=${TARGET_PXC_VERSION} \
+  --set pxc.datadir.size=${PXC_DATADIR_SIZE} \
+  --set pxc.replicas=${REPLICAS} \
+  --set deployerHelm.image="gcr.io/cloud-marketplace-tools/k8s/deployer_helm:0.8.0" \
+  --set operator.serviceAccountName=${OPERATOR_SERVICE_ACCOUNT} \
+  --set CDRJobServiceAccount=${CRD_SERVICE_ACCOUNT} \
+  > ${APP_INSTANCE_NAME}_manifest.yaml
+```
+
+#### Apply the manifest to your Kubernetes cluster
+
+Use `kubectl` to apply the manifest to your Kubernetes cluster:
+
+```shell
+kubectl apply -f "${APP_INSTANCE_NAME}_manifest.yaml" --namespace "${NAMESPACE}"
+```
+
+#### View the app in the Google Cloud Platform Console
+
+To get the Console URL for your app, run the following command:
+
+```shell
+echo "https://console.cloud.google.com/kubernetes/application/${ZONE}/${CLUSTER}/${NAMESPACE}/${APP_INSTANCE_NAME}"
+```
+
+To view the app, open the URL in your browser.
+
+### Access Percona XtraDB Cluster within the network
+
+You can connect to Percona XtraDB Cluster without exposing it to public access, using the
+`mysql` command line interface. We recommend making all connection via the corresponding network service.
+
+The user credentials are configured on application start up by the operator. More information about user management
+could be found [here](https://www.percona.com/doc/kubernetes-operator-for-pxc/users.html)
+
+#### Connect to Percona XtraDB Cluster using a client Pod
+
+Run percona-client and connect its console output to your terminal (running it may require some time to deploy the correspondent Pod):
+
+```shell
+kubectl run -i --rm --tty percona-client --image=percona:8.0 --restart=Never \
+            --env MYSQL_PASSWORD=$(kubectl get secret/${APP_INSTANCE_NAME}-secrets -o jsonpath='{.data.root}' | base64 -d) \
+            --env NAMESPACE=${NAMESPACE} \
+            --env NAME=${APP_INSTANCE_NAME} \
+            -- bash -il
+#wait for shell and than run
+mysql -h ${NAME}-haproxy -uroot -p${MYSQL_PASSWORD}
+```
+
+# Scaling
+
+## Scaling the cluster up or down
+
+By default, Percona Server for MongoDB application is deployed using 3 replicas.
+To change the number of replicas, use the following command:
+
+```
+kubectl patch psmdb "${APP_INSTANCE_NAME}" \
+  --namespace "${NAMESPACE}" \
+  --type=json \
+  -p '[{"op":"replace","path":"/spec/replsets/0/size","value":'${REPLICAS}'}]'
+```
+
+Where `REPLICAS` is the number of replicas you want.
+
+# Backup and Restore
+
+The detailed set of steps for backup and restores you can find here [Backups and Restores documentation](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/backups.html).
+
+# Uninstall the Application
+
+## Using the Google Cloud Platform Console
+
+1. In the GCP Console, open [Kubernetes Applications](https://console.cloud.google.com/kubernetes/application).
+
+1. From the list of applications, click **Percona Server MongoDB Operator**.
```

Comment:
Not sure how it would look for PXC, but needs to be changed.

Reply:
done